src/schema/opportunity.ts

```diff
@@ -471,6 +471,11 @@ export const typeDefs = /* GraphQL */ `
     """
     location: String
 
+    """
+    Whether the location is verified (from dataset_location or custom) or inferred from geo flags
+    """
+    locationVerified: Boolean
```

Comment:
Another thought, should we consider an enum here for future facing? (non blocking)

Reply:
enum as in? I think it can only be true/false right (it's technically user-provided not user-verified)

What would you say the enum would hold?

Reply:
Good idea, i'll do that quickly